popup-portfolio.js

```diff
@@ -0,0 +1,261 @@
+    title: 'Keeping track of hundreds of components',
+    techs: ['Ruby on Rails', 'css', 'JavaScript'],
+    techsdesktop: ['Codekit', 'GitHub', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    content: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+    seelive: 'See Live',
+    seesource: 'See Source',
+  },
+
+  {
+    id: 'popup-project-5',
+    closeBut: 'images/cancelicon.png',
+    closedesktop: 'images/closedesktop.png',
+    image: 'images/portfolioimg.png',
+    imageportfolio: 'images/imageportfolio.png',
+    title: 'Keeping track of hundreds of components',
+    techs: ['Ruby on Rails', 'css', 'JavaScript'],
+    techsdesktop: ['Codekit', 'GitHub', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    content: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+    seelive: 'See Live',
+    seesource: 'See Source',
+  },
+
+  {
+    id: 'popup-project-6',
+    closeBut: 'images/cancelicon.png',
+    closedesktop: 'images/closedesktop.png',
+    image: 'images/portfolioimg.png',
+    imageportfolio: 'images/imageportfolio.png',
+    title: 'Keeping track of hundreds of components',
+    techs: ['Ruby on Rails', 'css', 'JavaScript'],
+    techsdesktop: ['Codekit', 'GitHub', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    content: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+    seelive: 'See Live',
+    seesource: 'See Source',
+  },
+];
+
+const popupArray = [];
+
+for (let i = 0; i < projects.length; i += 1) {
+  const mediaQuery = window.matchMedia('(max-width: 768px)');
+  if (mediaQuery.matches) {
+    const popup = document.createElement('div');
+    popup.classList.add('closingpopup');
+    popup.classList.add('popuppt');
+
```

Comment:
- [ ] Kindly follow the design layout of the components in mobile view. For example the buttons should be at the bottom of the pop up and the technologies below the buttons should not overflow out of the container. Please fix this.
- [ ] Also center the image for the pop-up so that it matches the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/217635077-2a429294-c8bc-4322-83d8-c12279f203da.png)

Given template
![image](https://user-images.githubusercontent.com/81932478/217635573-f8d8d469-7249-47dd-ac20-6e2e53aad8f5.png)